__tests__/app.test.js

```diff
@@ -197,12 +199,71 @@ describe('GET /api/articles/:article_id/comments', () => {
             body: 'Awesome Article'
           };
         return request(app)
-          .post('/api/articles/998/comments')
+          .post('/api/articles/887/comments')
           .send(newComment)
           .expect(500)
-          .then((response) => {
-            expect(response.body.msg).toBe('Internal Server Error');
+          .then(({body}) => {
+            expect(body.msg).toBe('Internal Server Error');
           });
-      });
+     });
 })
 
+describe('PATCH /api/articles/:article_id', () => {
+    test('PATCH: 200 responds with the updated article', () => {
+        const newVote = {
+            inc_votes: 200
+        }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(newVote)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.updatedArticle).toMatchObject({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: 300,
+                article_img_url: expect.any(String)
```

Comment:
We could be more specific with all the values here, this will give us confidence that our patch hasn't impacted any of the other properties.